website/src/components/DataTable/DataTable.tsx

```diff
@@ -59,7 +61,7 @@ export type DataTableProps<T> = {
   filterValues?: FilterItem[];
   onNextClick?: () => void;
   onPreviousClick?: () => void;
-  onFilterChange?: (items: FilterItem[]) => void;
+  onFilterChange?: (items: FilterItem[], filterByColumn?: string) => void;
```

Comment:
Why do you need filterByColumn here? Isn't `id` in `FilterItem` enough?

Reply:
I went and removed it